requirements.txt

```diff
@@ -1,5 +1,5 @@
 --extra-index-url https://aibs.jfrog.io/aibs/api/pypi/ni-pypi-local/simple
-allensdk<=2.3.2; python_version >= '3.0'
+allensdk
```

Comment:
why not specify version 2.4.0 here?

Reply:
The idea was to unpin it so that the newest version will install. Since 2.4.0 is currently the version on pip, it'd be installed. We could say >=2.4.0 to make it clear that it's no longer compatible with earlier versions.

Reply:
I just changed requirements to read 'allensdk>=2.4.0'